src/components/Filters/Filters-container.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid/Grid';
+import clsx from 'clsx';
+
+import useFiltersStyles from './Filters-styles';
+
+const Filters = () => {
+  const classes = useFiltersStyles();
+  const [activeBlockchainButtons, setBlockchainButtons]:[ Record<string, boolean>, Function] = useState({
+    'BTC': false,
+    'ETH': true,
+    'XRP': false,
+    'LTC': false,
+    'BCH': false,
+    'XTZ': true,
+    'ADA': false,
+    'EOS': false,
+    'XLM': false
+  });
+
+  const filterHandler = (label: string) => {
+    if (label === 'ETH' || label === 'XTZ') {
+      const buttons = activeBlockchainButtons;
+      buttons[label] = !buttons[label];
```

Comment:
This is mutating your state through means other than `set` hook. I am actually surprised it is not throwing any warnings or errors. On line 23 you are assigning a REFERENCE of your state variable `activeBlockchainButtons` to `buttons` and on line 24 the mutation happens. This is anti-functional approach.

I would create a deep copy of your object, in this case it would be `const buttons = {...activeBlockchainButtons}` and it would protect you from mutating your state possibly in the future in multiple places (impossible to reason about and very easy to create hard to find bugs).